tests/puppeteer/campaigns/commonTests/loginBO.js

```diff
@@ -0,0 +1,13 @@
+const {expect} = require('chai');
```

Comment:
Does this works if it's outside of the export ?

Reply:
yes